maps/src/G3SkyMap.cxx

```diff
@@ -636,6 +850,10 @@ PYBINDINGS("coordinateutils") {
 	      "together.  Returns a new map object.  Map dimensions must be a "
 	      "multiple of the rebinning scale.")
 
+	    .def("compress", &G3SkyMap::Compress, (bp::arg("zero_nans")=false),
+	      "Convert the map to its default sparse representation, excluding "
+	      "empty pixels, and optionally converting NaN values to zeroes.")
```

Comment:
Can you include some verbiage here about when you might want/not want to turn NaNs to zeros? E.g. "Converting NaNs to zeroes will reduce memory use, but will remove the distinction in unweighted (or weight-removed) maps between unobserved regions and regions with zero temperature."

Reply:
fixed in 5546cb